.github/workflows/docs.yml

```diff
@@ -37,3 +37,14 @@ jobs:
 
       - name: Build HTML documentation
         run: make docs-html
+
+  vale:
```

Comment:
I am uncertain of whether this will work. I recall I tried this for `plone/documentation`, and I forgot the reason it did not work. Possibly I had to pass in an environment variable to tell Vale which files to inspect, which this GitHub Action does not support? If it does not work, then some version of this might work:

https://github.com/plone/documentation/blob/e83a788bf49083213456b28a222d85d1c9a9df53/.github/workflows/test.yml#L28-L33

Also I am not sure how to test this while developing. Perhaps you can modify this path while testing, and remember to yank it before merge? Give it a shot to see what happens. It's just a pull request at this time.

```yml
  pull_request:
    paths:
      - 'docs/**'
      - 'github/workflows/docs.yml'
```

Reply:
> Possibly I had to pass in an environment variable to tell Vale which files to inspect, which this GitHub Action does not support?

I think you can [give an optional input](https://github.com/errata-ai/vale-action#files-default-all) to this GitHub action to tell vale which files to lint. Please let me if I am getting it wrong😅

Reply:
Give it a try, and let's see!

Reply:
We definitely need to configure it to find Markdown files. Here's the latest report:

https://github.com/plone/volto/runs/11539747236

Nothing found. :sad: